### DISCUSSION TITLE
Change Top Solid Layers at Z Height

### DISCUSSION THREAD

**Discussion by swpaxton66** [supermerill/SuperSlicer#3448] [Q&A] [answered]
Hello All, 

Love Super Slicer!

I want to be able to change the number of top solid layers at a Z height value.  Is that possible?  I can't seem to find that setting / possibility.  

For example in Simplify3D I can create a second process and change the settings there. I can set at which Z height each process is applied.  

Thanks!

**Comment by neophyl** [comment, marked as answer]
Yes you can do that.  Like most things with different settings you use a modifier.  The simplest one to use as you mention different heights is to use a height range modifier.  
You can then add various options to the modifier.  When selected there is Perimeters & Shell and Infill.  Those contain the commonly used options for those but there is also an Add Settings option which has a vastly expanded range of settings you can modify.  

There are several types of modifiers and there are limits on what options are available to the different types.  Also some combinations can cause unexpected results due to the way PS/SuSi process modifiers.  

You can right click on an object to add a modifier.  If you add a Height Range one then once added it will appear in the right hand object panel.  You can edit the height at which it starts and stops there.  You can also click on the cog icon in the object panel next to the height modifier to add settings to it.

While it looks slightly different to Prusa Slicer (and the options available are many more than PS) you might want to look at their Prusa Slicer knowledge base https://help.prusa3d.com/article/modifiers_1767

**Reply by swpaxton66** [reply]
Thank you for the speedy and detailed reply!  I see now I am good.  I just needed a nudge in the right direction.  Take care, be safe :-)
